codalab/lib/download_manager.py

```diff
@@ -303,6 +307,9 @@ def netcat(self, uuid, port, message):
         return bytestring
 
     def _is_available_locally(self, target):
+        file_path = self._get_target_path(target)
+        if file_path.startswith("azfs://"):
```

Comment:
Maybe we should rename `_is_available_locally` since AzureFS is definitely not local (is available externally or directly)?